### ISSUE TITLE
Can't run examples with three.js r51 and r52

### ISSUE THREAD

**Issue by RobbieOnGithub** [zz85/sparks.js#22]
I'd love to use this library, but it seems that it isn't fully compatible with the latest versions of three.js.  I was wondering if an updated version of the library will come out - and if so - when?

**Comment by zz85** [comment]
sparks.js is compatible with the latest versions of three.js - although examples should get some updating.

for the syntax with the latest three.js - you can check out http://mrdoob.github.com/three.js/examples/webgl_particles_shapes.html

**Comment by RobbieOnGithub** [comment]
Yup yup - I was just looking at this today.  Great library btw - nice job.
